src/daemon.rs

```diff
@@ -112,6 +112,30 @@ struct NetworkInfo {
     relayfee: f64, // in BTC
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+struct TxIndexInfo {
+    synced: bool,
+    best_block_height: usize,
+}
+
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "lowercase")]
+enum IndexInfo {
+    TxIndex(TxIndexInfo),
+}
+
+#[derive(Copy, Clone)]
+enum TxIndexState {
+    /// Didn't check yet, re-check next time
+    Unknown,
+    /// assume unavailable but re-check after a minute since `last_check`
```

Comment:
This comment also says after a minute while the code waits an hour. They should be the same once we agree on the period.

Reply:
Gonna make it 10 minutes !